packages/compose/src/higher-order/with-safe-timeout/index.tsx

```diff
@@ -66,13 +68,14 @@ const withSafeTimeout = createHigherOrderComponent< TimeoutProps >(
 			}
 
 			render() {
-				const props = {
-					...this.props,
-					setTimeout: this.setTimeout,
-					clearTimeout: this.clearTimeout,
-				} as TProps;
-
-				return <OriginalComponent { ...props } />;
+				return (
+					// @ts-ignore
```

Comment:
can we see if we can get rid of this or replace it with `ts-expect-error` (or whatever the expect-error syntax is?)